src/atomate2/abinit/jobs/base.py

```diff
@@ -116,13 +167,16 @@ class BaseAbinitMaker(Maker):
         Keyword arguments that will get passed to :obj:`.run_abinit`.
     task_document_kwargs : dict[str, Any]
         Keyword arguments that will get passed to :obj:`.TaskDoc.from_directory`.
+    stop_jobflow_on_failure : bool
+        If True, stop all other jobs of the flow. False by default.
     """
 
     input_set_generator: AbinitInputGenerator
     name: str = "base abinit job"
     wall_time: int | None = None
     run_abinit_kwargs: dict[str, Any] = field(default_factory=dict)
     task_document_kwargs: dict[str, Any] = field(default_factory=dict)
+    stop_jobflow_on_failure: bool = False
```

Comment:
let's use the same convention as for stop_children in vasp so a function should_stop_jobflow and stop_jobflow_kwargs instead of the bool stop_jobflow_on_failure